app/views/search/search.html.erb

```diff
@@ -60,7 +60,12 @@
             <p><%= truncate(strip_tags(d.summary), length: 200, separator: ' ') %></p>
           </div>
         <% end %>
-        <%= paginate @datasets %>
+        <div class="pagination">
+          <nav>
+            <%= page_entries_info @datasets, entry_name: 'dataset' %>
+            <span class="paginate"><%= paginate @datasets, class:'paginate' %></span>
+          </nav>
```

Comment:
For accessibility reasons, it would be good to have invisible text for screen reader, in front of each link number, eg `<a href="..."><span class="visuallyhidden">Page </span>1</a>`. Is it possible with Kaminari?